src/esp/assets/ResourceManager.cpp

```diff
@@ -2234,7 +2239,6 @@ void ResourceManager::initDefaultMaterials() {
   shaderManager_.set<gfx::MaterialData>(WHITE_MATERIAL_KEY, whiteMaterialData);
   auto* perVertexObjectId = new gfx::PhongMaterialData{};
   perVertexObjectId->perVertexObjectId = true;
-  perVertexObjectId->vertexColored = true;
```

Comment:
Interesting. 
Then the instance mesh will not be displayed properly because before we set this vertexColored in the "material" to info the simulator that any mesh chunk that uses this material is vertexColored.

InstanceMesh is such a case. (mesh_semantic.ply in any Replica dataset model, in `/habitat/` sub-folder)